cfme/networks/provider/nuage.py

```diff
@@ -44,6 +45,30 @@ class events(Tab, BeforeFillMixin):
         validate = Button('Validate')
 
 
+class ValidateStatsMixin(object):
```

Comment:
I think we introduced this mixin in one of the other PRs, no need to copy-paste it with this commit